stable-diffusion-xl-1.0/config.yaml

```diff
@@ -12,16 +12,24 @@ model_name: Stable Diffusion XL
 model_framework: custom
 model_type: custom
 python_version: py39
+hf_cache:
+- repo_id: stabilityai/stable-diffusion-xl-base-1.0
+  ignore_patterns:
+  - diffusion_pytorch_model.safetensors
+  - text_encoder/model.safetensors
+  - text_encoder_2/model.safetensors
+  - unet/diffusion_pytorch_model.safetensors
+  - vae/diffusion_pytorch_model.safetensors
 requirements:
 - transformers
 - accelerate
 - safetensors
 - diffusers
 - invisible-watermark>=0.2.0
 resources:
-  accelerator: A10G
+  accelerator: A100
```

Comment:
@varunshenoy does this need A100? I think ideally models in truss-examples don't require it  (since not all users will have access)

Reply:
hopfully very soon we can enable it for everyone btw.